tutorials/create-an-example-application.md

```diff
@@ -0,0 +1,473 @@
+# Create an example application
+
+In the following steps, we will gradually build a minimal `go-dqlite` application that provides an HTTP endpoint to read and write values in a Dqlite cluster.
+
+[note type="important" status="Info"]
+ You can find the final code here: https://github.com/canonical/go-dqlite/blob/master/cmd/dqlite-demo/dqlite-demo.go  <!-- wokeignore:rule=master --> 
+[/note]
+
+## Create a minimal starting point
+
+We can now begin building our example application. In your project folder, create a new file called `go-dqlite-demo.go` and place the following code in it:
+
+```Go
```

Comment:
Can you run `go fmt` on all the go code you produce please?

Reply:
Done